README.md

```diff
@@ -1,10 +1,204 @@
+
+## Getting Started
+
+### Prerequisites
+
+- Python 3.10 or higher
+- pip or conda package manager
+
+### Installation
+
+#### Basic Installation (Python Library)
+
+```
+# Make sure pip is up to date
+python -m pip install -U pip
+
+# Install planetcantile
+python -m pip install planetcantile
+```
+
+#### Installation with Web Application
+
+```
+# Install with web application dependencies
+python -m pip install "planetcantile[app]"
+
+# Launch the application
+python -m uvicorn planetcantile.app:app --host 0.0.0.0 --port 8000
+```
+
+#### Installation from Source
+
+```
+# Clone the repository
+git clone https://github.com/AndrewAnnex/planetcantile.git
+cd planetcantile
+
+# Install with web application (Includes the core library plus dependencies)
+pip install -e ".[app]"
+
+# Or install for development (Includes all dependencies plus development tools)
+pip install -e ".[dev]"
+
+# Then launch the application (with auto-reload for development)
+uvicorn planetcantile.app:app --host 0.0.0.0 --port 8000 --reload
+```
+
+#### Docker Container (Experimental)
+
+Warning: The Docker container is currently experimental and may crash unexpectedly. It can be useful for quick exploration but is not recommended for production use: [planetcantile_docker](https://github.com/AndrewAnnex/planetcantile_docker)
+
+## Usage
+
+### Option 1: Using the Web API
+
+After starting the web application, the API interface will be accessible at [http://localhost:8000/docs](http://localhost:8000/docs)
+
+More information about endpoints is available in the [TiTiler documentation](https://developmentseed.org/titiler/).
+
+Example tile request:
+
+```
+http://localhost:8000/cog/tiles/MarsGeographicSphere/{z}/{x}/{y}?url=https://path/to/mars.tif
+```
+
+For local files, replace the URL with the local file path: ``url=file:///home/user/path/to/mars.tif``.
+
+### Option 2: As a Python Library
+
+
+```python
+# Import the planetary TMS collection
+from planetcantile import planetary_tms
+
+# List available TMS
+print(planetary_tms.list())
+
+# Get a specific TMS
+mars_tms = planetary_tms.get("MarsGeographicSphere")
+
+# Register a custom TMS (example)
+# planetary_tms.register(custom_tms)
+
+# Use with morecantile
+from morecantile import Tile
+tile = Tile(x=0, y=0, z=0)
+bounds = mars_tms.bounds(tile)
+print(f"Bounds of Mars tile: {bounds}")
+```
+
+### Environment Variables
+
+<strong>TILEMATRIXSET_DIRECTORY</strong>
+This environment variable allows you to specify a directory containing custom TileMatrixSet JSON definitions. Planetcantile will merge these with its own TMS definitions.
+
+You can set this programmatically before importing planetcantile:
+
+```python
+import os
+
+# Set the environment variable
+os.environ["TILEMATRIXSET_DIRECTORY"] = "/path/to/your/custom/tms/definitions"
+
+# Import planetcantile - it will include your custom TMS
+from planetcantile import planetary_tms
```

Comment:
actually for this, you no longer need to use planetcantile for the import, you can just use the morecantile library directly.

Reply:
err to expand on this, you may need to add a 1 liner to discover the path to planetcantile's TMSs with the python's site-packages folder. I think you could use something like sysconfig to grab it(https://stackoverflow.com/questions/122327/how-do-i-find-the-location-of-my-python-site-packages-directory), but this is more for cases when users don't want to use planetcantile's app or the planetcantile import for the tms directory.